index.js

```diff
@@ -184,6 +207,12 @@ var Umzug = module.exports = redefine.Class({
     }
   },
 
+  log: function() {
+    if (this.options.logging) {
+      this.options.logging.apply(null, [].slice.call(arguments));
```

Comment:
understood and agreed. let's simplify for now and add more arguments, such as a possible options arguments, only when actually needed